skymp5-functions-lib/src/sweetPie.ts

```diff
@@ -180,9 +184,13 @@ export class SweetPieGameModeListener implements GameModeListener {
         }
       }
     } else if (dialogId === this.leaveRoundConfirmationDialog[0]) {
+      const roundIndex = getPlayerCurrentRoundIndex(this.rounds, actorId);
```

Comment:
maybe better `thid.rounds.indexOf(getPlayerCurrentRound)` to not to create extra function